aspnetcore/host-and-deploy/proxy-load-balancer.md

```diff
@@ -210,6 +210,32 @@ services.Configure<ForwardedHeadersOptions>(options =>
 });
 ```
 
+### Configuration for an IPv4 address represented as an IPv6 address
+
+If the server is using dual-mode sockets, IPv4 addresses are supplied in an IPv6 format (for example, `10.0.0.1` in IPv4 represented in IPv6 as `::ffff:10.0.0.1` or `::ffff:a00:1`). See [IPAddress.MapToIPv6](xref:System.Net.IPAddress.MapToIPv6*). Determine if this format is required by looking at the [HttpContext.Connection.RemoteIpAddress](xref:Microsoft.AspNetCore.Http.ConnectionInfo.RemoteIpAddress*).
+
+In the following example, a network address that supplies forwarded headers is added to the `KnownNetworks` list in IPv6 format:
+
+Short format IPv6 for `10.11.12.1/8`:
+
+* `::ffff:0a0b:0c01`
```

Comment:
`::ffff:10.11.12.1`

Reply:
I think so ... There's already the little note for +96, so it's an easy matter to note down there that the hex address works, too. I'll ping u on the PR.